addons/godot-xr-tools/player/player_body.tscn

```diff
@@ -3,16 +3,14 @@
 [ext_resource path="res://addons/godot-xr-tools/player/player_body.gd" type="Script" id=1]
 
 [sub_resource type="CapsuleShape" id=1]
-radius = 0.4
+radius = 0.2
```

Comment:
this might be by accident

Reply:
owh this is because of the changed default on the parent, so I guess this is correct after all?

Reply:
Changed it and the comment :)